mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -46,6 +49,18 @@ def test_brainvision_data_filters():
     assert_equal(raw.info['highpass'], 0.1)
     assert_equal(raw.info['lowpass'], 250.)
 
+def test_brainvision_data_software_filters_latin1_global_units():
+    """Test reading raw Brain Vision files
+    """
```

Comment:
We recently settled on the convention of having this on the same line like `"""blah blah"""` but haven't converted all our files yet, but please change this one